lib/file.rb

```diff
@@ -0,0 +1,15 @@
+require 'csv'
+#this class creates a FileData object which is an object that has csv_file and
+# you can call read method(s) on this object 
+class FileData
```

Comment:
I like the idea of moving all this to its own class. You might be particularly interested in the last part of the FarMar live code video at https://adaacademy.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=2c336ca7-b2a8-4a78-941b-cd297639b17d